### PR TITLE
ENH Sphinx website [merge?]

### LINKED ISSUE #214: Enable texext in Sphinx
Enable the [texext](https://pypi.org/project/texext/) extension in Sphinx and convert all occurences of `` :math:`...` `` to `$...$`.

### PR DESCRIPTION
**Pull request by edouardoyallon**
Close #214 

- [x] Pretty website
- [x] Consistent website with the README.md
- [x] Clearer menu
- [x] Use of customization tools(such as github star count, etc)
- [x] Support of the DNS
- [x] Support smartphones navigation
- [x] Google analytics
- [x] Texext
- [x] Adding twitter

### CODE REVIEW

**Comment by eickenberg** [comment]
fyi I rebased this PR to incorporate all the other doc changes that have happened since, and I pushed the docs built from this PR to the website.

Take a look and when you have an update on this, feel free to push again

**Comment by eickenberg** [comment]
when you get a chance this weekend to add the logo that would be great.

Was there anything else you wanted to change?

**Comment by edouardoyallon** [comment]
> Was there anything else you wanted to change?

I haven't addressed yet each points of the check list above. Just as an example, the index.html of the website is not really convincing and I intend to modify it when I'll get a chance to have a bit more time.(you can save me some if you feel to rebasing..) Sorry for the delay.

**Comment by edouardoyallon** [comment]
Please check and review: https://www.kymat.io/

**Comment by lostanlen** [comment]
This is fantastic.
One important details: could you use a sans serif font?

**Comment by edouardoyallon** [comment]
Going to propose a new one! I'll also add a "Contact" button that will act as follow: when the user clicks on it, a message will display "You understand that questions regarding the software or its usage should not be sent to this email yet they should be reported to the github issue tracker of kymat.io?"

**Comment by edouardoyallon** [comment]
I'm happy with the results; I added a twitter logo, a github logo, as well as an iOS/favicon logo, I change the font, it looks quite pro! I deployed it successfully online!

Please merge ASAP to avoid conflicts with future modifications!

**Comment by lostanlen** [comment]
thanks. can you do 
`font-family:"Helvetica Neue",Helvetica,Arial,sans-serif` ?

**Comment by janden** [comment]
So the currently deployed website is what's in this PR?

File: doc/source/conf.py [hunk 1]
```diff
@@ -103,7 +108,21 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {
+    'logo': 'kymatio.jpg',
```
Comment (janden) [inline doc/source/conf.py]: Thought we were going to do SVG here?

Comment (edouardoyallon) [reply doc/source/conf.py]: @janden the JPG one is 16kb so.. way lighter!

**Comment by edouardoyallon** [comment]
Fixed! Can you merge?

File: kymatio/scattering3d/scattering3d.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ def _rotation_covariant_convolution_and_modulus(self, input_array, l, j):
             tensor of the same size as input_array. It holds the output of
             the operation::
 
-            .. math::   \\sqrt{\\sum_m (\\text{input}_\\text{array} \\star \\psi_{j,l,m})^2)}
+            $\sqrt{\sum_m (\text{input}_\text{array} \star \psi_{j,l,m})^2)}
```
Comment (eickenberg) [inline kymatio/scattering3d/scattering3d.py]: Close the math expression?

Comment (eickenberg) [reply kymatio/scattering3d/scattering3d.py]: 💵 💲 💰

File: doc/source/index.rst [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
    sphinx-quickstart on Tue Oct  2 23:41:09 2018.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
 
-Kymatio: wavelet scattering in Python
-*************************************
-kymatio is a Python module for computing wavelet and scattering transforms.
+
+Kymatio:
```
[the remark this answers is not in the document]
Comment (edouardoyallon) [reply doc/source/index.rst]: solved

File: doc/source/quickstart.rst [hunk 4]
```diff
@@ -1,9 +1,7 @@
 Quick Start
 ===========
 
-On Linux or macOS, open a shell and run::
-
-    pip install kymatio
+On Linux or macOS, open a shell and run the instruction of https://github.com/kymatio/kymatio .
```
[the remark this answers is not in the document]
Comment (edouardoyallon) [reply doc/source/quickstart.rst]: OK.

File: kymatio/scattering1d/scattering1d.py [hunk 5]
```diff
@@ -23,31 +23,28 @@ class Scattering1D(object):
     alternated with a complex modulus non-linearity. The scattering transform
     of a 1D signal :math:`x(t)` may be written as
 
-        :math:`S_J x = [S_J^{(0)} x, S_J^{(1)} x, S_J^{(2)} x]`
+        $S_J x = [S_J^{(0)} x, S_J^{(1)} x, S_J^{(2)} x]$
 
     where
 
-        :math:`S_J^{(0)} x(t) = x \\star \\phi_J(t)`,
+        $S_J^{(0)} x(t) = x \star \phi_J(t)$,
```
[the remark this answers is not in the document]
Comment (edouardoyallon) [reply kymatio/scattering1d/scattering1d.py]: no.

**Comment by edouardoyallon** [comment]
AFAIK, "\" is not necessary anymore.

**Comment by eickenberg** [comment]
merged.

Everybody: there is a lot of $$-math changes in different places in docstrings. If you plan to modify docstrings in a PR, please rebase first

File: doc/source/conf.py (hunk 1 above)
Comment (janden) [reply doc/source/conf.py]: :rofl:

**Pull request merged**